app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (<h1>Hello World!</h1>);
```

Comment:
- [ ] Please `Application` component add a route that leads to the `Greeting` component, this will make your Greeting component to be shown on your app 👍🏿

Reply:
@prolajumokeoni done